app/src/components/common/layout/header/index.tsx

```diff
@@ -238,13 +257,26 @@ const HeaderContainer: React.FC = (props: any) => {
     onClick: () => history && history.push('/'),
   }
 
+  const governanceMarketProps = {
+    onClick: () => {
+      setGovernancePage(!governancePage)
+      setMarketPage(!marketPage)
+    },
+  }
+
```

Comment:
I would pass this onClick inline to the component directly....And then setState based on the one you clicked...

Reply:
👍